### PR TITLE
Add man pages

### PR DESCRIPTION
**Pull request by mrunalp**
Signed-off-by: Mrunal Patel <mrunalp@gmail.com>

### CODE REVIEW

**Comment by mikebrow** [comment]
Looks good.  Maybe a `README.md`?

**Comment by mrunalp** [comment]
Added README.md

**Comment by mrunalp** [comment]
@crosbymichael @LK4D4 @hqhq PTAL

**Comment by hqhq** [comment]
Some format issues when I apply this PR:
```
$ git apply a.patch
a.patch:64: trailing whitespace.

a.patch:78: trailing whitespace.

a.patch:82: trailing whitespace.
"ubuntu01" removing "ubuntu01" from the runc list of containers:
a.patch:83: trailing whitespace.

a.patch:106: trailing whitespace.

warning: squelched 19 whitespace errors
warning: 24 lines add whitespace errors.
```

File: man/runc-checkpoint.8.md [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+
+Where "<container-id>" is the name for the instance of the container to be
+checkpointed.
+
+# DESCRIPTION
+   The checkpoint command saves the state of the container instance.
+
+# OPTIONS
+   --image-path 		path for saving criu image files
+   --work-path 			path for saving work files and logs
```
Comment (hqhq) [inline man/runc-checkpoint.8.md]: Lots of format issues like this, maybe we should use spaces instead of tabs?

**Comment by hqhq** [comment]
Minor format issues, other than that, LGTM.

**Comment by rajasec** [comment]
@mrunalp 
Are you planning to add apparmor & process in exec usage as per #619 or you will handle in different PR.

File: man/runc-checkpoint.8.md (hunk 1 above)
Comment (mrunalp) [inline man/runc-checkpoint.8.md]: Updated to convert tabs to spaces.

**Comment by mrunalp** [comment]
Updated to convert tabs to spaces and included updates to runc exec.

**Comment by hqhq** [comment]
LGTM

**Comment by mrunalp** [comment]
@dqminh @LK4D4 @crosbymichael PTAL

**Comment by crosbymichael** [comment]
LGTM

**Pull request merged**